workers/data_refinery_workers/downloaders/test_sra.py

```diff
@@ -80,12 +82,48 @@ def test_download_file_ncbi(self):
         assoc.save()
         result, downloaded_files = sra.download_sra(dlj.pk)
         utils.end_downloader_job(dlj, result)
-        self.assertTrue(result)
-        self.assertEqual(downloaded_files[0].sha1, "e7ad484fe6f134ba7d1b2664e58cc15ae5a958cc")
-        self.assertTrue(os.path.exists(downloaded_files[0].absolute_file_path))
+
+        # If the FTP server is down, then we expect that the downloader job should have failed
+        ftp_server = "ftp.sra.ebi.ac.uk"
+        ftp = FTP(ftp_server)
+        if ftp.login()[0:3] == "550":
```

Comment:
Oh nice, checking for the failure in the tests! I like that, it'll make them less brittle!